jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -0,0 +1,109 @@
+lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
+
+pipeline {
+    agent none
+    environment {
+        AWS_ROLE_ARN = "arn:aws:iam::${AWS_ACCOUNT_PUBLIC}:role/opensearch-test"
+        AWS_ROLE_SESSION_NAME = "jenkins-test-session"
+    }
+    tools {
+        jdk "JDK14"
```

Comment:
Lets create an issue; should we only run tests with JDK14? I think this will influence OpenSearch to be running on that jdk version which we added support to run on jdk11

Reply:
Good point, I've created #1495